proxy/backendratelimit_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build !race || redis
```

Comment:
Interestingly the new syntax was introduced to avoid confusion between && and ||. Now it is clear that it is OR and I am not sure that `// +build !race redis` is actually correct, maybe it should be `// +build !race,redis` which would translate into `//go:build !race && redis`

Reply:
Old syntax ref https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md#background-syntax

Reply:
I think `//go:build !race && redis` makes sense.

Reply:
I am wondering do we need `redis` tag at all?

Reply:
not everyone needs to have a local redis running

Reply:
The point is since it is `//go:build !race || redis` these tests were always executed during normal (not -race) tests regardless of redis tag

Reply:
> not everyone needs to have a local redis running

It requires redis binary as we start new redis server for each test
https://github.com/zalando/skipper/blob/6b089867410bf4b9f56b4686450cf95301ac02c7/net/redistest/redistest.go#L14

Reply:
I think what we want here is `//go:build !race && redis`

Reply:
We can merge like this since it is equivalent to the existing setup to unblock other builds and reconsider redis condition separately

Reply:
https://github.com/zalando/skipper/issues/1854